example/invited_talks.yml

```diff
@@ -0,0 +1,8 @@
+- id: 1
```

Comment:
ID will be matched to a .txt file that has `abstract` and `bio` sections, which are latex.